dev/com.ibm.ws.cdi.shared.weld/bnd.bnd

```diff
@@ -49,16 +49,10 @@ Import-Package: !org.scannotation.archiveiterator, \
                 !sun.nio.cs, \
                 !service, \
                 !sun.security.util, \
-                javax.persistence;version="[1,3)", \
-                org.jboss.weld.*;version="[2,4)", \
-                com.ibm.ws.cdi.executor; version="1.0.0", \
-                com.ibm.ws.cdi.utils; version="1.0.0", \
-                javax.enterprise.context; version="[1.1.0,3.0.0)", \
-                javax.enterprise.context.spi; version="[1.1.0,3.0.0)", \
-                javax.enterprise.event; version="[1.1.0,3.0.0)", \
-                javax.enterprise.inject; version="[1.1.0,3.0.0)", \
-                javax.enterprise.inject.spi; version="[1.1.0,3.0.0)", \
-                javax.enterprise.util; version="[1.1.0,3.0.0)", \
+                javax.persistence;version="[1.1,3)", \
+                org.jboss.weld.*;version="[2.4.3,4.0.0)", \
+                javax.enterprise.context.*; version="[1.1.0,3.0.0)", \
+                javax.enterprise.inject.*; version="[1.1.0,3.0.0)", \
```

Comment:
This is wrong. Don't expose too many packages. Please just stay with the previous set.

Reply:
These are imports, not exports ... and there's actually less than before (isn't there?!).